koku/api/report/test/oci/test_oci_query_handler.py

```diff
@@ -1597,11 +1600,12 @@ def test_multi_exclude_functionality(self, _):
                 exclude_one = None
                 exclude_two = None
                 for exclude_option in opt_list:
-                    if "no-" not in exclude_option.get(ex_opt):
+                    _exclude_option = exclude_option.get(ex_opt)
```

Comment:
Default this to string to avoid an `AttributeError` if `ext_opt` isn't in `exclude_option`.
```suggestion
                    _exclude_option = exclude_option.get(ex_opt, "")
```